.ci/pipeline/coverity.yml

```diff
@@ -0,0 +1,27 @@
+variables:
+  COVERITY_TOOL_HOME: $(Agent.BuildDirectory)/cov-analysis
+  DESCRIPTION: Nightly
```

Comment:
I would put some additional info here  as for now we will have all scan with same description


![image](https://user-images.githubusercontent.com/18015221/110833579-a4287500-82ad-11eb-9775-3aa4fccd822a.png)

Reply:
But it has different upload date and proposed to used only for nightly runs for a while.